core-blocks/list/index.js

```diff
@@ -80,7 +98,11 @@ export const settings = {
 			},
 			{
 				type: 'raw',
-				isMatch: ( node ) => node.nodeName === 'OL' || node.nodeName === 'UL',
+				selector: 'ol,ul',
+				schema: {
+					ol: listContentSchema.ol,
+					ul: listContentSchema.ul,
+				},
```

Comment:
Should just be `schema: listContentSchema`, no?

Reply:
Hm, no, it would contain phrasing content too. List content can contain phrasing content and lists.